### ISSUE TITLE
SDK shouldn't provide any dependency in alpha version

### ISSUE THREAD

**Issue by hatched-mariusz** [RadiusNetworks/flybuy-android#17]
It is hidden problem cased by `com.google.maps:google-maps-services` dependency. 

In version `com.radiusnetworks.flybuy:pickup:2.7.0` you update the `com.google.maps:google-maps-services` dependency from version `2.0` to version `2.1` and `com.google.maps:google-maps-services:2.1.0` added a new dependency: 
`com.squareup.okhttp3:okhttp:5.0.0-alpha.7` because in my application I using `com.squareup.okhttp3:okhttp:4.9.3`

Now I'm getting runtime errors - HTTP-400. 
It is important to don't introduce in SDK alpha dependencies. 
Now I can exclude okhttp group from pickup like that:
```
api ('com.radiusnetworks.flybuy:pickup') // If using pickup feature
    {
        exclude group: "com.squareup.okhttp3", module: "okhttp"
    }
```
but I'm not sure how much `com.google.maps:google-maps-services` dependency base on okhttp and if I will not get another runtime error. 
Please back to `com.google.maps:google-maps-services:2.0.0` or use 2.1.0 but exclude okhttp dependency and test your sdk, or take version 2.6.3 and merge the solution for concurrent modification exception and generate version 2.6.4

**Comment by botnerd** [comment]
It looks like this was fixed in [Google Maps Services SDK v2.1.1](https://github.com/googlemaps/google-maps-services-java/releases/tag/v2.1.1). We're working on an update to bump that dependency version.

**Comment by hatched-mariusz** [comment]
Glad U find a solution 
Please generate a new build because this issue generates for Us the most crashes: 
<img width="1268" alt="Screenshot 2022-12-09 at 09 10 33" src="https://user-images.githubusercontent.com/47123235/206655815-fc0a958d-931a-4a20-bb7d-4a1e6a95435d.png">

**Comment by botnerd** [comment]
This has been fixed in the [2.7.4 release](https://github.com/RadiusNetworks/flybuy-android/releases/tag/2.7.4). If you run into any issues, let us know.
